pwa/src/components/soaps/soapForm.js

```diff
@@ -322,28 +321,32 @@ export default function SoapsForm({id}) {
                               title: "RequestSkeleton",
                               id: "requestSkeletonAccordion",
                               render: function () {
-                                return <ElementCreationNew id="requestSkeleton" label="RequestSkeleton" data={soap?.requestSkeleton} />
+                                return <ElementCreationNew id="requestSkeleton" label="RequestSkeleton"
+                                                           data={soap?.requestSkeleton}/>
                               }
                             },
                               {
                                 title: "RequestHydration",
                                 id: "requestHydrationAccordion",
                                 render: function () {
-                                  return <ElementCreationNew id="requestHydration" label="RequestHydration" data={soap?.requestHydration} />
+                                  return <ElementCreationNew id="requestHydration" label="RequestHydration"
+                                                             data={soap?.requestHydration}/>
```

Comment:
Hier is niets veranderd. alleen de layout van de code is veranderd. Komt dit doordat je een prettier aan hebt staan? Misschien goed om te kijken wat we hiermee kunnen doen.

Nu komen namelijk allemaal changes in git naar boven die zorgen voor ruis.